rero_ils/modules/ill_requests/api.py

```diff
@@ -60,7 +64,15 @@ def get_ill_requests_total_for_patron(self, patron_pid):
         :param patron_pid: the patron pid being searched.
         :return: return total of ill requests.
         """
-        return self.filter('term', patron__pid=patron_pid).count()
+        months = current_app.config.get('RERO_ILS_ILL_HIDE_MONTHS', -6)
```

Comment:
Maybe adapt the docstring to give the info that the total includes only last six months (total active ill requests).